### DISCUSSION TITLE
Me trying to bring people from discord to here.

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#236] [General]
![mariachi simulación!](https://whatsondisneyplus.com/wp-content/uploads/2021/02/so_104_still-01-1-1-1024x576.jpg)

**Comment by jlm4487** [comment]
Do I have to run "sudo github" to migrate from the Discord?

**Reply by riffnshred** [reply]
DONT RUN SUDO 👀. sudo is for admin access. super secure !!!!

**Reply by jlm4487** [reply]
But how will I know if I've migrated properly?? I know, I will ask Claude Julien, he has plenty of time to teach me now.

**Reply by riffnshred** [reply]
lmao.

**Comment by riffnshred** [comment]
those who knows will understand. watch that show !!!! (solar opposites)

**Comment by xTrackstar04x** [comment]
The way to do it is to set up a feed. Every time that an New Issue/ New Discussion gets started it sends the link to Discord (and obviously make it read only like the "announcement")

**Reply by riffnshred** [reply]
Thats not a bad idea ! Ill look into this

**Comment by xxxDrewedxxx** [comment]
```git reset --hard
git checkout GitHub
git pull
chmod +x scripts/installGitHub.sh
./scripts/install.sh

rm -f Discord
